tests/Doctrine/Tests/ORM/Tools/Pagination/LimitSubqueryOutputWalkerTest.php

```diff
@@ -208,37 +207,7 @@ public function testCountQueryWithArithmeticOrderByCondition()
         $query->setHint(Query::HINT_CUSTOM_OUTPUT_WALKER, 'Doctrine\ORM\Tools\Pagination\LimitSubqueryOutputWalker');
 
         $this->assertSame(
-            'SELECT DISTINCT id_0, (1 - 1000) * 1 FROM (SELECT a0_.id AS id_0, a0_.name AS name_1 FROM Author a0_) dctrn_result ORDER BY (1 - 1000) * 1 DESC',
```

Comment:
Revert them for now- leaving on vacation today, returning Feb. 1- I'll
devise a fix then. Would appreciate some guidance per my comment in the
failing testcase PR thread. Ciao--
On Jan 24, 2015 7:15 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In
> tests/Doctrine/Tests/ORM/Tools/Pagination/LimitSubqueryOutputWalkerTest.php
> <https://github.com/doctrine/doctrine2/pull/1283#discussion_r23496003>:
>
> > @@ -208,37 +207,7 @@ public function testCountQueryWithArithmeticOrderByCondition()
> >          $query->setHint(Query::HINT_CUSTOM_OUTPUT_WALKER, 'Doctrine\ORM\Tools\Pagination\LimitSubqueryOutputWalker');
> >
> >          $this->assertSame(
> > -            'SELECT DISTINCT id_0, (1 - 1000) * 1 FROM (SELECT a0_.id AS id_0, a0_.name AS name_1 FROM Author a0_) dctrn_result ORDER BY (1 - 1000) * 1 DESC',
>
> @zeroedin-bill <https://github.com/zeroedin-bill> what should I do about
> these tests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/doctrine2/pull/1283/files#r23496003>.
>